lib/process-services-cloud/src/lib/form/components/widgets/date/date-cloud.widget.ts

```diff
@@ -71,8 +70,7 @@ export class DateCloudWidgetComponent extends WidgetComponent implements OnInit,
 
     dateInputControl: FormControl<Date> = new FormControl<Date>(null);
 
-    private onDestroy$ = new Subject<void>();
-
+    private destroyRef = inject(DestroyRef);
```

Comment:
please group it together with rest private readonly fields when you change it to private readonly